source/install/transport-encryption/config-database.rst

```diff
@@ -0,0 +1,118 @@
+=============================================
+Configuring Database Transport Encryption
+=============================================
+
+Mattermost is able to encrypt the traffic between the database and the application
+using TLS. This guide will walk you through the set up steps for a single, separate MySQL
+server.
+
+## Prerequisites
+
+- Mattermost server or cluster operational
+- MySQL server operational
+- Connectivity between Mattermost and MySQL server confirmed
+- Authentication credentials for Mattermost user on MySQL server
+
+Example Environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+In this scenario there is 1 Mattermost application server and one MySQL server,
+both running Ubuntu 18.04, with the following IPs:
+
+- **transport-encryption-mattermost1:** 10.10.250.146
+- **transport-encryption-mysql1:** 10.10.250.148
+
+Preparations
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- Connect to both servers with a sudo or root user
+
+MySQL Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+On the MySQL server, execute the following command to prepare the server for
```

Comment:
```suggestion
Execute the following command to prepare the server for
```
Removed the first bit because it's in the MySQL section and seems redundant. 3/5.